d2/src/main/java/com/linkedin/d2/xds/GlobCollectionUtils.java

```diff
@@ -1,41 +1,77 @@
 package com.linkedin.d2.xds;
 
+import javax.annotation.Nullable;
+
 public class GlobCollectionUtils
 {
-  private static final String D2_URI_NODE_GLOB_COLLECTION_PREFIX = "xdstp:///indis.D2URI/";
+  private static final String D2_URIS_PREFIX = "/d2/uris/";
+  private static final String D2_URI_NODE_GLOB_COLLECTION_PREFIX = "xdstp:///indis.D2URI/d2/uris/";
```

Comment:
where does this come from? Isn't d2 uri urn of the format   xdstp:///indis.D2URI/{CLUSTER}/{URL_ESCAPED_ANNOUNCER}?

Reply:
Yeah this is an old version, I was messing around with stuff. This is gone